docs/platforms/apple/guides/ios/session-replay/customredact.mdx

```diff
@@ -72,6 +73,36 @@ To hide the username, we need to mask it.
         .sentryReplayMask()
     }
     .background(.blue)
-    .sentryReplayUnmask() 
+    .sentryReplayUnmask()
+  }
+```
+
+## Debugging Session Replay masking
+
+To see how elements are being masked, enable the masking preview from anywhere in your app. It will display a semi-transparent overlay on top of the masked elements.
+This works on the simulator and on device, as well as within Xcode Preview.
```

Comment:
```suggestion
To see how elements are being masked, enable the masking preview from anywhere in your app. It will display an overlay on top of the masked elements. This works on the simulator and on device, as well as within Xcode Preview.
```

Reply:
since by default it's not semi-transparent 😅